--- test/functional/fixtures/api/es-next/assertions/test.js	2021-05-04 19:05:02.381617700 +0000
+++ test/functional/fixtures/api/es-next/assertions/test.js	2021-05-04 19:56:41.381617700 +0000
@@ -299,9 +299,11 @@
         expect(getSnapshotWarnings()[0]).contains("> 253 |        await t.expect(await Selector('#el1').innerText).eql('');");
     });
 
-    it.skip('Should raise multiple warnings when awaiting multiple selector properties in one assertion', async function () {
+    it.only('Should raise multiple warnings when awaiting multiple selector properties in one assertion', async function () {
         await runTests('./testcafe-fixtures/assertions-test.js', 'Multiple awaited selector properties in one assertion', { only: 'chrome' });
 
+        console.log(testReport.warnings);
+
         expect(getSnapshotWarnings().length).to.eql(2);
         expect(getSnapshotWarnings()[0]).contains("> 259 |    await t.expect(await selector.innerText + await selector.innerText).eql('');");
         expect(getSnapshotWarnings()[1]).contains("> 259 |    await t.expect(await selector.innerText + await selector.innerText).eql('');");
